docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -42,11 +49,25 @@ Alternatively, you can also watch our Grafana for Beginners series where we disc
 - [Docker Compose](https://docs.docker.com/compose/) (included in Docker for Desktop for macOS and Windows)
 - [Git](https://git-scm.com/)
 
-### KillerCoda sandbox environment (Alternative)
+{{% /class %}}
```

Comment:
What does this do? You will need to wrap this one in an ignore as the transformer won't know how to handle this yet

Reply:
that puts the text into a grey box. it was already there
i will remove it 👍🏼